src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java

```diff
@@ -21,31 +20,31 @@
 
 public class MessageWall extends WikiBasePageObject {
 
-  final By firstMessageWrapperBy =
+  final static By firstMessageWrapperBy =
       By.cssSelector(".comments li.SpeechBubble.message.message-main:nth-child(1)");
-  final By replyButtonBy = By.cssSelector(".replyButton");
-  private final By messageTitleBy = By.cssSelector(".msg-title");
-  private final By messageBodyBy = By.cssSelector(".msg-body");
-  private final By imageBy = By.cssSelector(".thumbimage");
-  private final By messageTextBoldBy = By.cssSelector("b");
-  private final By messageTextItalicBy = By.cssSelector("i");
-  private final By messageTextBy = By.cssSelector(".msg-body *");
-  private final By messageLinkBy = By.cssSelector("a");
-  private final By messageUserNameBy = By.cssSelector(".edited-by > a:nth-child(1)");
-  private final By moreButtonBy = By.cssSelector(".wikia-menu-button.secondary.combined");
-  private final By editButtonBy = By.cssSelector(".edit-message");
-  private final By removeButtonBy = By.cssSelector(".remove-message");
-  private final By reopenButtonBy = By.cssSelector(".reopen-thread");
-  private final By quoteButtonBy = By.cssSelector(".quote-button.secondary");
-  private final By quoteMessageBy = By.cssSelector(".replies p");
-  private final By saveChangesButtonBy = By.cssSelector(".save-edit");
-  private final By closeThreadInfobox = By.cssSelector(".deleteorremove-bubble > .message");
-  private final By replyBodyBy = By.cssSelector(".replyBody");
-  private final String newMessageMenu =
+  final static By replyButtonBy = By.cssSelector(".replyButton");
+  private final static By messageTitleBy = By.cssSelector(".msg-title");
+  private final static By messageBodyBy = By.cssSelector(".msg-body");
+  private final static By imageBy = By.cssSelector(".thumbimage");
+  private final static By messageTextBoldBy = By.cssSelector("b");
+  private final static By messageTextItalicBy = By.cssSelector("i");
+  private final static By messageTextBy = By.cssSelector(".msg-body *");
+  private final static By messageLinkBy = By.cssSelector("a");
+  private final static By messageUserNameBy = By.cssSelector(".edited-by > a:nth-child(1)");
+  private final static By moreButtonBy = By.cssSelector(".wikia-menu-button.secondary.combined");
+  private final static By editButtonBy = By.cssSelector(".edit-message");
+  private final static By removeButtonBy = By.cssSelector(".remove-message");
+  private final static By reopenButtonBy = By.cssSelector(".reopen-thread");
+  private final static By quoteButtonBy = By.cssSelector(".quote-button.secondary");
+  private final static By quoteMessageBy = By.cssSelector(".replies p");
+  private final static By saveChangesButtonBy = By.cssSelector(".save-edit");
+  private final static By closeThreadInfobox = By.cssSelector(".deleteorremove-bubble > .message");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)